src/content/blog/2024-08-05-rafiki-code-architecture.mdx

```diff
@@ -0,0 +1,203 @@
+---
+layout: ../../layouts/BlogLayout.astro
+title: "Architecture Breakdown of Rafiki (Pending Title)"
+description: 
+date: 2024-08-05
+slug: rafiki-low-level-intro
+author: Nathan Lie
+author_url: https://www.linkedin.com/in/nathan-lie-138a73121
+tags:
+  - Interledger
+  - Rafiki
+---
+import LargeImg from '/src/components/pages/LargeImg.astro'
+
+## Introduction
+
+It has been said that the Interledger Foundation upholds open technology as one of its core values. It has also been said that [Rafiki](https://github.com/interledger/rafiki), a white-label application that enables Interledger usage, upholds this value by being open-source and allowing anyone to contribute.
+
+It has _not_ been said, however, how one comes to intimately understand Rafiki. It's not impossible, to be sure, but the Interledger endeavor has been around for long enough that a write-up at a lower level is warranted.
+
+So, what makes Rafiki tick? What do we see when we venture beyond _what_ Rafiki does and into _how_ it does those things? 
+
+### Disclaimer
+This article assumes the reader has high-level knowledge of the following concepts:
+
+* [Open Payments](https://openpayments.dev/introduction/overview/)
+* [Interledger](https://interledger.org/developers/get-started/#how-does-interledger-work)
+* [Rafiki](https://rafiki.dev/introduction/overview/)
```

Comment:
Isn't it weird that this is a post on Rafiki and it assumes pre-knowledge on Rafiki? I know you are saying that it assumes high level pre-knowledge, but can we maybe phrase it a bit differently here?